django/utils/deprecation.py

```diff
@@ -113,19 +119,9 @@ def __repr__(self):
             ),
         )
 
-    def _async_check(self):
```

Comment:
I changed this because I didn’t want to create an instance variable outside of `__init__`. I couldn’t find a reason for a separate method or any references in docs.

(It seems the separate method actually contributed to a bug: https://code.djangoproject.com/ticket/31928.)